core/src/test/scala/org/apache/spark/scheduler/DAGSchedulerSuite.scala

```diff
@@ -1376,17 +1376,25 @@ class DAGSchedulerSuite
   }
 
   test("misbehaved resultHandler should not crash DAGScheduler and SparkContext") {
+// The job will be failed on first task throwing a DAGSchedulerSuiteDummyException.
```

Comment:
The comments should still be indented. You may just need to wrap the lines to stay under 100 characters.